tests/DpBadge.spec.js

```diff
@@ -0,0 +1,44 @@
+import { shallowMount } from '@vue/test-utils'
+import DpBadge from '~/components/DpBadge/DpBadge.vue'
+
+describe('DpBadge', () => {
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = shallowMount(DpBadge, {
+      props: {
+        color: 'default',
+        size: 'medium',
+        text: 'Test Badge'
+      }
+    })
+  })
+
+  it('renders the correct text', () => {
+    expect(wrapper.text()).toBe('Test Badge')
+  })
+
+  it('applies the correct color class', () => {
+    expect(wrapper.classes()).toContain('color-text-default')
+    expect(wrapper.classes()).toContain('bg-color-light')
+  })
```

Comment:
Maybe we could create a test for each value that the color prop can have? And we could adjust the code like this:

```suggestion
it('applies the correct colors for color=default', () => {
  const defaultClasses = ['color-text-default', 'bg-color-light']
  defaultClasses.forEach(class => {
    expect(wrapper.classes()).toContain(class)
  })
})
```

Then we could remove the "renders with different color" test below